### PR TITLE
CIP-maat

### PR DESCRIPTION
**Pull request by aleeusgr**
Questions and comments are very welcome.
The current blocker is the lack of self-balancing and credible reputation metrics based on well-defined inputs other than subjective ratings.
Thank you!

### CODE REVIEW

**Review by RSoulatIOHK — COMMENTED** [review]
There are a few typos here and there but it's not really important. I feel like I get the gist of the flow to create and burn the tokens. 
However, I am not an expert on the matter so I'm not sure how practical or efficient this solution is. I'm not even fully sure I fully get which problem this is really solving. 

Finally, I am not sure it fits the Certification working group, but this is only my personal opinion.

File: Agendas/CIP-Maat.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+To solve this two issues I propose CIP-Maat: a self balancing incentive framework.
+
+I want to utilize a private blockchain and two tokens. 
+One token is native to the private blockchain and is called the proof of guilt, or the Hype token.
+
+The second token is called the proof of shame, or the Cooldown token. It is a Cardano Native Asset.
+
+The Cooldown token is dispensed (pseudo)-randomly to a certain fraction of the whole population of dReps. 
+What it does is it shows of the users accounts, and it is in essence a warning ticket. The holder of the token is the wallet to which it was minted, and its effect is that it reduces voting power by 1/n^2, where n is the number of tokens the user has recieved.
```
[the remark this answers is not in the document]
Comment (aleeusgr) [reply Agendas/CIP-Maat.md]: Yeah, sorry. This does not make sense.

I read it back and you are right to point out that we should emphasize tokens minting and burning as a mechanic and I missed that at all

**Review by aleeusgr — COMMENTED** [review]
you rightly point out that I missed opportunities to involve tokens.

If I am honest, I do not have one coherent idea. I am trying to address the challenge of the Summit Hackathon 2023: Transparency and Governance.

One way, described earlier, was to threaten the delegates, volunteers, with terminating their role if they do not look at the community feedback when assigned the duty.

This is fundamentally wrong in current state of Cardano, as the pressure is the opposite from what I imagined: we should attract people to be dReps, not seek reasons to ban them.

The question remains "what is the problem with transparency?" and my guess is 

"How can we make a working group treasury more transparent so that the thing can work by itself and require as little top down control as possible?"

**Comment by aleeusgr** [comment]
Yeah, and you are right that it is beyond the scope of the group.
Conversing with CIP editors I got the idea that it would be good to present this idea here for your expertise in formal systems.

**Pull request closed without merging**